install-devenv.sh

```diff
@@ -38,12 +38,7 @@ echo "${sha256}  ${devenv_cache}/${archive}" | /usr/bin/shasum -a 256 --check --
 tar --strip-components=1 -C "$devenv_python_root" -x -f "${devenv_cache}/${archive}"
 
 if ! [[ -d "${devenv_root}/devenv" ]]; then
-    if [[ $CI ]]; then
-        git -C "$devenv_root" clone -q -b "$branch" --depth=1 'https://github.com/getsentry/devenv.git'
-    else
-        # This makes sure we clone over ssh so that github keys are working.
-        git -C "$devenv_root" clone -q -b "$branch" --depth=1 'git@github.com:getsentry/devenv'
-    fi
+    git -C "$devenv_root" clone -q -b "$branch" --depth=1 'https://github.com/getsentry/devenv.git'
```

Comment:
Cloning over https will work at this point as it's unauthenticated. When I wrote the previous code, `devenv` was a private repo at the time.